fetch.py

```diff
@@ -16,7 +16,7 @@
 import requests
 import openstack
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger()
```

Comment:
`main` has been removed but now `root` is there, which tells even less. I suggest to set it to "imagebuilder"